packages/wdio-appium-service/src/launcher.ts

```diff
@@ -173,7 +173,6 @@ export default class AppiumLauncher implements Services.ServiceInstance {
             } else if (error) {
                 errorMessage += `\n${error.toString()}`
             }
-            log.error(errorMessage)
```

Comment:
I think we should not remove it but instead check if error code is !== 0 and only log then

Reply:
sure, make sense.